style.css

```diff
@@ -133,343 +110,146 @@ section {
   list-style-type: none;
 }
 
-/* secttion two area */
-.section-two {
-  background-image: none;
-  width: 327px;
+/* The section area */
+
+.case1 {
+  display: flex;
+  justify-content: center;
+  padding: 50px 20px;
+  background-color: #e5e5e5;
 }
 
-.section-two .background {
-  border-radius: 10px;
-  box-shadow: 3px 3px 8px #888;
-  margin-left: 20px;
-  margin-right: 20px;
-  height: 500px;
-  padding-left: 10px;
-  padding-top: 15px;
-  background: #fff;
+.case1 .case2 {
+  padding: 20px;
+  background-color: #fff;
+  border-radius: 20px;
 }
 
-.section-two .background div img {
-  width: 263px;
-  height: 220px;
+.case1 .case2 ul {
+  display: flex;
 }
 
-.about-myself-container h2 {
-  margin-right: 50px;
-  margin-left: 20px;
-  font-style: normal;
+.about-section h2 {
   font-weight: 700;
   font-size: 40px;
   line-height: 52px;
-  width: 263px;
+  color: #172b4d;
+  margin-bottom: 20px;
 }
 
-.section-two .background h2 {
-  font-style: normal;
+.case1 .case2 h2 {
   font-weight: 700;
   font-size: 32px;
   line-height: 44px;
-  margin-top: 5px;
-  margin-bottom: 5px;
   color: #172b4d;
+  margin-top: 20px;
+  margin-bottom: 20px;
 }
 
-.about-myself-container ul {
-  margin-left: -27px;
-  margin-top: 10px;
-}
-
-.section-two .background ul {
+.about-section div ul li {
   display: flex;
+  justify-content: space-between;
 }
 
-.about-myself-container ul li {
-  padding-right: 4px;
+.about-section div ul ul li {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  background-color: #f7f7f9;
+  margin-bottom: 15px;
+  padding: 10px;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
 }
```

Comment:
- [ ] Kindly make sure to the same typography styles (text color, font-weight, etc....) used to display the languages, languages, and skills in order to meet the provided Figma design. :wink: 

   - ### Your design
    ![image](https://user-images.githubusercontent.com/87197412/199302260-0e1821dd-bc70-405c-a4a2-60253b758d47.png)


   - ### Figma design
    ![image](https://user-images.githubusercontent.com/87197412/199302465-c6d5c79d-3f58-4164-8287-ba47cf923869.png)